honeybee_doe2/writer.py

```diff
@@ -21,7 +21,12 @@ def model_to_inp(hb_model):
     comp_data = ComplianceData()
     sb_data = sbd()
 
-    hb_model.convert_to_units(units='Feet')
+    hb_model = hb_model.duplicate()
+
+    if hb_model.units != 'Feet':
+        hb_model.convert_to_units(units='Feet')
+    hb_model.remove_degenerate_geometry()
+    hb_model.rectangularize_apertures(subdivision_distance=0.5, max_separation=0.1)
```

Comment:
I feel like people are really going to want to change these values and we should probably expose them on the function and in the CLI. Or, if you don't want to expose it, I think a default max_separation of 0 is going to be a more reliable default.

Reply:
Thanks! I set the max_separation to 0 for now. I think we only want to expose this when people start asking for it.